packages/react-querybuilder/src/utils/formatQuery/defaultValueProcessorByRule.ts

```diff
@@ -1,24 +1,34 @@
 import type { ValueProcessorByRule } from '../../types/index.noReact';
 import { toArray, trimIfString } from '../arrayUtils';
 import { isValidValue, quoteFieldNamesWithArray, shouldRenderAsNumber } from './utils';
+import { defaultQuoteChar } from '../../defaults';
 
-const escapeSingleQuotes = (
+const escapeStringValueQuotes = (
   v: string | number | boolean | object | null,
+  quoteChar: string,
   escapeQuotes?: boolean
-) => (escapeQuotes && typeof v === 'string' ? v.replaceAll(`'`, `''`) : v);
+) =>
+  escapeQuotes && typeof v === 'string'
+    ? v.replaceAll(`${quoteChar}`, `${quoteChar}${quoteChar}`)
```

Comment:
In your third-party SQL library, does a string like `"inner""quotes"` work the same as `'inner''quotes'` would in other SQL processors? Or does it require a different escape character like `\`?

Reply:
Yes it works the same. I mean it's not about the processor but these quotes are supported by MySQL or MariaDB and they behave the same as single quotes afaik https://mariadb.com/kb/en/string-literals/

Reply:
Ok, that makes sense now. Thanks.